covalent/_file_transfer/enums.py

```diff
@@ -49,6 +51,7 @@ class FileTransferStrategyTypes(str, enum.Enum):
     "http": FileTransferStrategyTypes.HTTP,
     "https": FileTransferStrategyTypes.HTTP,
     "s3": FileTransferStrategyTypes.S3,
+    "gs": FileTransferStrategyTypes.GCloud,
```

Comment:
@wjcunningham7 I think I can now better articulate my earlier comment on the Azure PR -- since the scheme is normally identified by the URL prefix (as done here for example), it is nonstandard to have "https" refer to both HTTPS and and Blob schemes.

Reply:
I agree with that but I don't know of a good solution that doesn't introduce something else nonstandard.